src/uct/api/uct.h

```diff
@@ -812,10 +813,10 @@ UCT_INLINE_API ucs_status_t uct_ep_put_short(uct_ep_h ep, const void *buffer, un
  * @brief
  */
 UCT_INLINE_API ucs_status_t uct_ep_put_bcopy(uct_ep_h ep, uct_pack_callback_t pack_cb,
-                                             void *arg, size_t length, uint64_t remote_addr,
```

Comment:
the user does not always know the length in advance. having a parameter named "length" may encourage UCT to mis-use it and to assume this is actually the length (rather than using the return value)